.changeset/gorgeous-gorillas-confess.md

```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-styles': patch
+---
+
+Fixed offset of radio button when checked and in inline layout.
```

Comment:
As the second problem is caused by the first and that the first was not announced in a previous changelog, I'll stick to this sentence about the first problem.